NTUEELightDance/LightDance-Editor
# EDITOR #162  Connect upload and download file api

### What is this PR for?

We now enable editors to download and upload export.json. 
We also check the format of export.json before upload to ensure that data in db is safe.
### What type of PR is it?
[Bug Fix | Improvement | Feature | Documentation | Hot Fix | Refactoring]
Feature 
### Todos
* [ ] - Beautify alert UI if wanted.

### What is the Github issue?

https://github.com/NTUEELightDance/LightDance-Editor/issues/162
### How should this be tested?

1. make some changes and save them. --> change 1
2. download export.json
3. maker other changes and save. --> change 2
4. upload export.json you just download.
5. refresh
6. you should see change 1 rather than change 2.
### Screenshots (if appropriate)

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No
